config/routes.rb

```diff
@@ -1,3 +1,5 @@
 Rails.application.routes.draw do
+  devise_for :users
   # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+  resources :posts, only: %i[new create index]
 end
```

Comment:
### [OPTIONAL]
This is a very good practice: Kindly add root route. When a user visits your application it is good to have a root route other than being redirected to the default " your on rails"  template

Use this [article](https://curriculum.railsbridge.org/intro-to-rails/setting_the_default_page) to learn more about this.